src/facade/reply_builder.cc

```diff
@@ -955,6 +948,10 @@ void RedisReplyBuilder2Base::SendVerbatimString(std::string_view str, VerbatimFo
   WritePieces(kCRLF);
 }
 
+std::string RedisReplyBuilder2Base::SerializeCommand(std::string_view command) {
+  return string{command} + kCRLF;
```

Comment:
Ignore me but small rant. I absolutely hate `+` on strings because is `not commutative` and it messes with generic algorithms and unfortunately even with `concepts` you can't really express this :disappointed: . That's the main reason I use `absl::StrCat`.